V2/gameFuseLeaderboardEntry.js

```diff
@@ -41,5 +41,44 @@ class GameFuseLeaderboardEntry {
             }, {});
         return dictionary;
     }
+
+    // method to get a user's full profile data
```

Comment:
If we are commenting this out but leaving it we should add a comment to why, or should we just delete the whole thing?

Reply:
I'll add a comment, I think it's nice to have the start of it rather than deleting it, I'm planning to add this method either this week or next week